R/view_config_val_errors.R

```diff
@@ -191,16 +171,16 @@ dataframe_to_markdown <- function(x) {
     gsub("[^']NA", "'NA'", .)
```

Comment:
Does this need to be modified for v4.0.0 so that it reads `null` or is this unrelated?

Reply:
It is related but does not need modifying. It's automatically substitues `NA`s as `'NA'`s  where necessary. It will just not be applied to `NULL` values in the new schema.